subprojects/docs/src/main/resources/header.html

```diff
@@ -165,50 +164,98 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#sample-gradle-builds" aria-expanded="false" aria-controls="sample-gradle-builds">Example Gradle Projects</a>
-                <ul id="sample-gradle-builds">
-                    <li><a href="https://github.com/gradle/gradle/tree/master/subprojects/docs/src/samples">Groovy DSL Samples</a></li>
-                    <li><a href="https://github.com/gradle/kotlin-dsl/tree/master/samples">Kotlin DSL Samples</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#controlling-transitive-dependencies" aria-expanded="false" aria-controls="controlling-transitive-dependencies">Controlling Transitives</a>
+                <ul id="controlling-transitive-dependencies">
+                    <li><a href="dependency_constraints.html">Declaring Constraints</a></li>
+                    <li><a href="platforms.html">Using Platforms</a></li>
+                    <li><a href="dependency_alignment.html">Aligning Dependencies</a></li>
+                    <li><a href="dependency_capability_conflict.html">Managing Conflicting Dependencies</a></li>
+                    <li><a href="component_metadata_rules.html">Fixing Metadata</a></li>
+                    <li><a href="resolution_rules.html">Customizing Resolution</a></li>
+                </ul>
+            </li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#modeling-features" aria-expanded="false" aria-controls="modeling-features">Utilizing Variants</a>
+                <ul id="modeling-features">
+                    <li><a href="component_capabilities.html">Declaring Capabilities</a></li>
+                    <li><a href="feature_variants.html">Modeling Feature Variants</a></li>
+                    <li><a href="variant_model.html">Understanding Variant Selection</a></li>
+                    <li><a href="cross_project_publications.html">Adding Variants to Subprojects</a></li>
+                </ul>
+            </li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#publishing" aria-expanded="false" aria-controls="publishing">Publishing Modules</a>
+                <ul id="publishing">
+                    <li><a href="publishing_setup.html">Setting up Publishing</a></li>
+                    <li><a href="publishing_gradle_module_metadata.html">Understanding Gradle Module Metadata</a></li>
+                    <li><a href="publishing_signing.html">Signing Artifacts</a></li>
+                    <li><a href="publishing_customization.html">Customizing Publishing</a></li>
+                </ul>
+            </li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#fine_grained_dm_apis" aria-expanded="false" aria-controls="fine_grained_dm_apis">Fine-grained Apis</a>
+                <ul id="fine_grained_dm_apis">
+                    <li><a href="dependency_modification_apis.html">Modifying Dependencies</a></li>
```

Comment:
I really wondered what this was about.

Proposal: Fine Tuning Dependencies

Reply:
Removed, and moved content elsewhere

Reply:
Removed, content moved elsewhere